app/com/linkedin/drelephant/tuning/hbt/ParameterGenerateManagerHBT.java

```diff
@@ -48,7 +48,7 @@ public ParameterGenerateManagerHBT(ExecutionEngine executionEngine) {
             TuningAlgorithm.OptimizationAlgo.HBT.name())
         // .eq(JobSuggestedParamSet.TABLE.isParamSetDefault, 0)
         .findList();
-    logger.debug(
```

Comment:
Why this change? This looks like an AutoTuning related change.

Reply:
Yes , this found out during last dr Elephant down , this should be info

Reply:
nit: If possible kindly raise another PR for this change, since it's a small change we can merge that PR in no time.

Reply:
Ok , i'll remove it from this . I agree , this is not related to current PR . Since it was minor and important change , i thought to do in current PR . I'll put in separate one